### PR TITLE
[1LP][RFR] Fix AttributeError on test_git_import module import

### PR DESCRIPTION
**Pull request by mshriver**
{{ pytest: cfme/tests/automate/test_git_import.py::test_automate_git_domain_removed_from_disk }}

### CODE REVIEW

File: cfme/tests/automate/test_git_import.py [hunk 1]
```diff
@@ -17,11 +17,19 @@
...
+
+try:
+    GIT_REPO_URL = cfme_data.automate_links.datastore_repositories.manageiq_automate
+except AttributeError:
+    pass
 
 
 @pytest.fixture
 def imported_domain(appliance):
+    if GIT_REPO_URL is None:
```
Comment (mshriver) [inline cfme/tests/automate/test_git_import.py]: Could also do this via a separate fixture instead of having to re-use the same block?

**Comment by dajoRH** [comment]
I detected some fixture changes in commit 2854dd4da84c4d308728c97bb118b9242ec25864

The *local* fixture **`imported_domain`** is used in the following files:
- **cfme/tests/automate/test_git_import.py**
    - *test_automate_git_domain_removed_from_disk*
    - *test_refresh_git_current_user*
    - *test_automate_git_verify_ssl*
    - *test_automate_git_import_deleted_tag*

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

**Comment by john-dupuy** [comment]
@mshriver Since this key, value pair is also in the template yamls, is it necessary to check this? https://github.com/ManageIQ/integration_tests/blob/813e26b28da94a156514f71b121d57c07072f106/conf/cfme_data.yaml.template#L682

**Comment by mshriver** [comment]
@john-dupuy I'd say yes - because its done on module import.

File: cfme/tests/automate/test_git_import.py [hunk 2]
```diff
@@ -17,11 +17,19 @@
...
+        pytest.skip('No automate repo url available at '
+                    'cfme_data.automate_links.datastore_repositories.manageiq_automate')
     repo = appliance.collections.automate_import_exports.instantiate(
```
Comment (ganeshhubale) [inline cfme/tests/automate/test_git_import.py]: **Optional:**
Can we modify this message:
```No automate datastore is available at URL - cfme_data.automate_links.datastore_repositories.manageiq_automate'```

**Pull request merged**